jscomp/gentype/ModuleExtension.ml

```diff
@@ -0,0 +1,28 @@
+open GenTypeCommon
+
+let shimExtension ~(config : Config.t) =
```

Comment:
The hope is to make shims unnecessary at some point.
Though for now, I guess worth adding to the changelog that the user needs to write the shim files extension accordingly now.

Reply:
This does not change the existing behavior. Users still write `.ts` shims regardless config.